src/analysis/normed_space/inner_product.lean

```diff
@@ -1824,15 +1838,65 @@ begin
   exact submodule.le_orthogonal_orthogonal ⊤
 end
 
-lemma submodule.eq_top_iff_orthogonal_eq_bot {K : submodule 𝕜 E} (hK : is_complete (K : set E)) :
-  K = ⊤ ↔ Kᗮ = ⊥ :=
+lemma submodule.eq_top_iff_orthogonal_eq_bot (hK : is_complete (K : set E)) : K = ⊤ ↔ Kᗮ = ⊥ :=
```

Comment:
I'd change the direction of this one, and label it `@[simp]`. Also, can you state the opposite lemma (`K\perp = top iff K = bot`)?